docs/3_finetuner_in_action/3_2_image_to_image.md

```diff
@@ -1,3 +1,154 @@
+
+Let's check the status of our runs.
+```python
+print(run.name, '-', run1.status())
+print(run.name, '-', run2.status())
+```
+
+```bash
+resnet-ttl-1 - {'status': 'CREATED', 'details': 'Run submitted and awaits execution'}
+resnet-ttl-2 - {'status': 'CREATED', 'details': 'Run submitted and awaits execution'}
+```
+
+## Reconnect and retrieve the runs
+Since some runs might take up to several hours/days, it's important to know how to reconnect to Finetuner and retrieve your runs.
+
+```python
+import finetuner
+finetuner.login()
+
+run1 = finetuner.get_run('resnet-ttl-1')
+run2 = finetuner.get_run('resnet-ttl-2')
+```
+
+You can monitor the runs by checking the status - `run.status()` or the logs - `run.logs()`. 
+
+If your runs have finished successfully, you can save fine-tuned models in the following way:
+```python
+run1.save_model('without-freezing')
+run2.save_model('freezed-model')
+```
+
+## Evaluation
+Currently, we don't have a user-friendly way to get evaluation metrics from the `EvaluationCallback` we initialized previously.
+What you can do for now is to call `run.logs()` in the end of the run and see evaluation results:
+
+```bash
+[10:37:49] DEBUG    Metric: 'model_average_precision' Value: 0.30105                                     __main__.py:217
+           DEBUG    Metric: 'model_dcg_at_k' Value: 0.43933                                              __main__.py:217
+           DEBUG    Metric: 'model_f1_score_at_k' Value: 0.06273                                         __main__.py:217
+           DEBUG    Metric: 'model_hit_at_k' Value: 0.69000                                              __main__.py:217
+           DEBUG    Metric: 'model_ndcg_at_k' Value: 0.43933                                             __main__.py:217
+           DEBUG    Metric: 'model_precision_at_k' Value: 0.03450                                        __main__.py:217
+           DEBUG    Metric: 'model_r_precision' Value: 0.30105                                           __main__.py:217
+           DEBUG    Metric: 'model_recall_at_k' Value: 0.34500                                           __main__.py:217
+           DEBUG    Metric: 'model_reciprocal_rank' Value: 0.30105                                       __main__.py:217
+           INFO     Done ✨                                                                              __main__.py:219
+           INFO     Saving fine-tuned models ...                                                         __main__.py:222
+           INFO     Saving model 'model' in /usr/src/app/tuned-models/model ...                          __main__.py:233
+           INFO     Pushing saved model to Hubble ...                                                    __main__.py:240
+[10:38:14] INFO     Pushed model artifact ID: '62a1af491597c219f6a330fe'                                 __main__.py:246
+           INFO     Finished 🚀                                                                          __main__.py:248
+```
```

Comment:
in general when you submit 2 runs, I'm expecting `run2` should be a bit more performant than `run1`, while only reporting one metric seems to be a bit confused.

Reply:
yeah, good point. Let's stick with one run then and make it two when we add a proper evaluation handling.